src/lib.rs

```diff
@@ -1,4 +1,7 @@
 //! A data type suitable for storing sensitive information such as passwords and private keys in memory, featuring constant time equality, mlock and zeroing out.
+extern crate core; // Needed by pre
+#[cfg(any(test, feature = "pre"))]
+extern crate pre;
```

Comment:
maybe it's time to migrate to 2018 edition..

Reply:
Do you want my to do that in this PR? It shouldn't be too much work.

Reply:
Yes, that's why I mentioned it :)